_components/button-group/button-group--default.md

```diff
@@ -15,4 +15,4 @@ variants:
     description: Display a group of buttons in a row as a single element.
 ---
 
-{% include component.html %}
+{% include component.html level="h3" %}
```

Comment:
The way the template is programmed, the default header is already`h2` and the exception is the value of `level`. We could switch the attribute to more of a binary like `h3_header: true` since `h3` is going to be the most common value for `level`, but I think we lose a bit of flexibility with that approach. I vote for leaving it as is. 

Also, let me know if I misunderstood!

Reply:
In that case, should we provide settings for any additional headers? With the current setup we can only accept the default `h2` or update to `h3` without it breaking logical structure. Updating to `h4` for example, will have third level headings nested beneath it.

Hopefully not overthinking or over-engineering here! Let me know what you think!